client/src/elements/schema-editor/schema-editor-table.js

```diff
@@ -45,18 +46,20 @@ function emptyToNull(data) {
 }
 
 @checkAvailability()
-@inject(Loader)
+@inject(Loader, I18N)
 export class SchemaEditorTable {
   @bindable schema;
   @bindable data;
   @bindable change;
 
+  minRowsDataTable = 8;
```

Comment:
could we add this to the editor config here https://github.com/nzzdev/Q-server-nzz/blob/dev/config/nzz/editor.js#L71 under a new key `table`? I am ok with defaulting to 8 in case the config is not given, but would love to have this configurable.
You can see this file for an example how to inject the config object and load config parameters from it: https://github.com/nzzdev/Q-editor/blob/master/client/src/elements/organisms/meta-editor.js

Reply:
Actually it would make more sense to have this configurable in the tool schema.json. This is then applied to the option property here: https://github.com/nzzdev/Q-editor/blob/dev/client/src/elements/schema-editor/schema-editor-table.js#L75. The default can be defined here: https://github.com/nzzdev/Q-editor/blob/dev/client/src/elements/schema-editor/schema-editor-table.js#L55